squad/frontend/templatetags/squad.py

```diff
@@ -196,3 +198,8 @@ def crispy(context, form, **options):
     for option, value in options.items():
         setattr(helper, option, value)
     return render_crispy_form(form, helper=helper, context=context)
+
+
+@register_global_function()
+def to_json(d):
+    return json.dumps(d)
```

Comment:
shouldn't you catch potential exception here?

Reply:
yeah, you're right. That's TypeError. I'll make it return an empty string if `d` can't be parsed to json